diff --git a/ports/curl/portfile.cmake b/ports/curl/portfile.cmake
index bdc544e9e..340d93865 100644
--- a/ports/curl/portfile.cmake
+++ b/ports/curl/portfile.cmake
@@ -64,6 +64,10 @@ if(VCPKG_TARGET_IS_WINDOWS)
     list(APPEND OPTIONS -DENABLE_UNICODE=ON)
 endif()
 
+if(VCPKG_TARGET_IS_OSX)
+    list(APPEND OPTIONS -DENABLE_IPV6=OFF)
+endif()
+
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS 
diff --git a/ports/curl/vcpkg.json b/ports/curl/vcpkg.json
index e028d3897..a63858e34 100644
--- a/ports/curl/vcpkg.json
+++ b/ports/curl/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "curl",
   "version": "8.4.0",
+  "port-version": 1,
   "description": "A library for transferring data with URLs",
   "homepage": "https://curl.se/",
   "license": "curl AND ISC AND BSD-3-Clause",
